### DISCUSSION TITLE
full screen problem

### DISCUSSION THREAD

**Discussion by AllenGo912** [Eugeny/tabby#10018] [Q&A]
Why does it always automatically go full screen? May I ask how to turn it off? I didn't find any relevant options in the settings

**Comment by AllenGo912** [comment]
I often tail - f to capture logs, and then it automatically becomes full screen, making it impossible to switch to other applications through the Windows taskbar

**Reply by Eugeny** [reply]
Did you accidentally enable docking in Settings -> Window by any change?

**Reply by AllenGo912** [reply]
Thank you for your reply, but I haven't opened this setting. I've been trying to reproduce this scenario recently. In the end, I think it might be caused by the full-screen hotkey.
![图片](https://github.com/user-attachments/assets/5f9f6e1c-6e98-410e-af8e-caf5397c8cc3)
![图片](https://github.com/user-attachments/assets/3d073bb5-e8bb-41cc-8164-07e86d3572c7)
